bc_obps/registration/api/user_operator.py

```diff
@@ -87,44 +87,74 @@ def save_operator(payload: UserOperatorOperatorIn, operator_instance: Operator,
         # create parent operator records
         operator_has_parent_operators: bool = payload.operator_has_parent_operators
         if operator_has_parent_operators:
-            po_operator_fields_mapping = {
-                "po_legal_name": "legal_name",
-                "po_trade_name": "trade_name",
-                "po_cra_business_number": "cra_business_number",
-                "po_bc_corporate_registry_number": "bc_corporate_registry_number",
-                "po_website": "website",
-            }
             for idx, po_operator in enumerate(payload.parent_operators_array):
```

Comment:
Since we don't use `idx` any longer, we don't need to use `enumerate` to get the index.